src/index.scss

```diff
@@ -0,0 +1,96 @@
+$main: rgba(0, 52, 0, 0.137);
```

Comment:
I'd like to have src/saas/variables.scss and have src/saas/animations.scss files. 

Please keep here only screen sizes like UI libs do. Specific styles should be stored in the related components.